docker-compose.yml

```diff
@@ -13,17 +13,17 @@ mysql:
     - MYSQL_DATABASE
     - MYSQL_USER
     - MYSQL_PASSWORD
-  ports:
-    - 3306:3306
+  expose:
```

Comment:
@ktbartholomew I see that there is an install guide to set up an env file and run a script that sources the environment. Would it be of interest to switch that in favor of the docker-compose.override.yml file? https://docs.docker.com/compose/extends/ This is automatically read by docker, but you still need to set up the file. It does not require an external script and does not require any additional tools by the dev.